### ISSUE TITLE
brickman stdout/err to journal

### ISSUE THREAD

**Issue by ensonic** [ev3dev/ev3dev#366]
Create this file.
```
# cat /etc/systemd/system/brickman.service.d/brickman.conf 
[Service]
Environment="G_MESSAGES_DEBUG=all"
StandardOutput=journal
StandardError=journal
```
Run ```systemctrl restart brickman.service``` and then ```journalctl -f -u brickman.service```. Now you'll see all glib warnings.

Ideally there are none, so we could consider doing this permanently. Any opinions.

**Comment by dlech** [comment]
I only use `G_MESSAGES_DEBUG=all` when actually debugging brickman. I don't see a need to output so many messages. By default, glib outputs errors which is what I have been using mostly in brickman anyway.

I don't have a problem with piping the brickman output to journal.

**Comment by ensonic** [comment]
Cool. I'll send a PR that adds the piping and I expand the README with the commands.

**Comment by dlech** [comment]
Tip:

If you add `ev3dev/ev3dev#366` to your commit messages, GitHub will automatically link the commits to this issue. I usually make the last line of my commit messages `Issue ev3dev/ev3dev#NNN`. If you write "fixes" or "closes" before the issue, it will automatically close the issue when the commit is merged, but I don't like doing that on ev3dev. I would rather have the issue open until the fix has been released, not just committed.

**Comment by ensonic** [comment]
Updated the ticket references to be "See issue ev3dev/ev3dev#xxx".

**Comment by dlech** [comment]
Awesome!

**Comment by dlech** [comment]
This has been released in brickman 0.6.2

**Comment by ensonic** [comment]
Thanks, confirming that it works.
